### ISSUE TITLE
Xdebug for vscode.dev

### ISSUE THREAD

**Issue by zobo** [xdebug/vscode-php-debug#704]
Research the idea of getting vscode-php-debug to work in the web version of vscode.

Docs provided @isidorn https://code.visualstudio.com/api/extension-guides/web-extensions

Since running PHP in the browser (start current script) isn't a real option and creating a TCP Listener socket is also out of the question (security sandbox) maybe websocket support in Xdebug.cloud #703 could be something to consider.

**Comment by isidorn** [comment]
Yes, I am not sure if this would be possible at all. However sounds like an interesting investigation.

**Comment by zobo** [comment]
Is it at all possible to have a debugger extension in vscode.dev?
The docs say
> If an extension wants to provide a debugger or terminal that also work in the web extension host, a browser entry point needs to be defined

So I guess, technically possible :)

**Comment by isidorn** [comment]
Yes, it is possible. One example is [mock-debug](https://github.com/microsoft/vscode-mock-debug) - which is a fake debugger, but still shows that it is technically possible.
